File: 0001-0500/0412_fizzBuzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        fizzbuzz = []
        for i in range(n):
            if (i+1) % 15 == 0:
                fizzbuzz.append('FizzBuzz')
            elif (i+1) % 3 == 0:
                fizzbuzz.append('Fizz')
            elif (i+1) % 5 == 0:
                fizzbuzz.append('Buzz')
            else:
                fizzbuzz.append(str(i+1))
        return fizzbuzz

File: 0001-0500/0412_fizzBuzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> output = new ArrayList<>();
        
        for (int i=1; i<n+1; i++) {
            if (i % 15 == 0) {
                output.add("FizzBuzz");
            } else if (i % 3 == 0) {
                output.add("Fizz");
            } else if (i % 5 == 0) {
                output.add("Buzz");
            } else {
                output.add(String.valueOf(i));
            }
        }
        
        return output;
    }
}
